imap/jmap_mail.c

```diff
@@ -9111,11 +9181,14 @@ static void _email_append(jmap_req_t *req,
     fclose(f);
     f = NULL;
 
+    jmap_set_emailid_ts(internaldate, detail->email_id);
```

Comment:
I would prefer the jmap_set_emailid functions to be more explicitly named: e.g. jmap_set_email_id_from_timespec and jmap_set_email_id_from_nanoepoch. I had to grep through to the code to figure when to pass what.

Reply:
sure.